app/helpers/application_helper.rb

```diff
@@ -1861,4 +1861,12 @@ def ext_auth?(auth_option = nil)
     auth_option ? ::Settings.authentication[auth_option] : true
   end
   public :ext_auth?
+
+  def translate_header_text(text)
```

Comment:
Not sure this makes to put this into application_helper.rb...just feels awfully global.  @martinpovolny Thoughts?